docs/user-guide/tool-integration/nuget-chocolatey-example.md

```diff
@@ -0,0 +1,305 @@
+# Nuget and Chocolatey
+
+This is an example of how to use the Strongbox artifact repository manager with Chocolatey using Nuget.
+
+## Pre-requisites
+
+Before you proceed please make sure you have installed and configured a [Strongbox Distribution](../getting-started.md).
+
+## Example project
+
+The "Hello, Strongbox!" example project can be found [here][hello-strongbox-nuget-chocolatey].
+
+## Installing choco
+
+<!-- todo: pymdownx.tabbed -->
+
+!!! success "Windows"
+    Chocolatey is available natively on Windows and can be installed following the [official documentation]
+
+!!! warning "Mono"
+    If you are running Linux/MacOS and would like to use [mono], there is no pre-built binary. 
+    You will need to head to the [chocolatey repository][choco-repo] for instructions on how to build it. 
+   
+!!! success "Linux / MacOS / Docker"    
+    We have already built a linux docker image with pre-built and configured [choco][choco] which we are also using 
+    in our integration tests. It is published at [strongboxci/alpine] and you can feel free to use it for development
+    purposes.
+
+It is OK to ignore warnings about a system reboot being requested since the nothing here is affected by it.
+
+## Configuring Choco
+
+### Get API key
+
+The NuGet protocol assumes that users need to be authenticated with `API Key` to be able to deploy or delete your packages.
+Strongbox provides the REST API to get an API Key for specified user:
+    
+``` tab="Windows" linenums="1"
+FOR /F %i IN ('curl -u admin http://localhost:48080/api/users/admin/generate-security-token') DO set API_KEY=%i
+echo %API_KEY%
+```
+
+``` tab="Linux" linenums="1"
+API_KEY=`curl -u admin http://localhost:48080/api/users/admin/generate-security-token`
+echo $API_KEY
+```
+
+Enter your Strongbox password. (Default is: admin/password)
```

Comment:
Suggestion: admin/password should be highlighted `admin/password`

Reply:
Good idea! Thanks!